content/blog/2023-03-17-build-grafana-dashboards-for-apache-skywalking-native-promql-support/index.md

```diff
@@ -0,0 +1,403 @@
+
+SkyWalking is support to ingest Prometheus metrics through [OpenTelemetry collector](https://opentelemetry.io/ecosystem/integrations/) and through the aggregate calculation of these metrics
+to provide a variety of systems monitoring, such as Linux Monitoring and Kubernetes monitoring.
+SkyWalking already provides [native UI](https://skywalking.apache.org/docs/main/next/en/ui/readme/) and [GraphQL API](https://skywalking.apache.org/docs/main/next/en/api/query-protocol/) for users.
+But as designed to provide wider ecological integration capabilities, since 9.4.0, it provides PromQL Service, the third-party systems or visualization platforms that already support PromQL (such as Grafana),
+could obtain metrics through it. SkyWalking users will benefit from it when they integrate with different systems.
+
+## What is PromQL Service in SkyWalking?
+PromQL Service is a query engine on the top of SkyWalking native GraphQL query, with additional query stage calculation capabilities powered by Prometheus expressions.
+It can accept PromQL HTTP API requests, parse Prometheus expression, and transform between Prometheus metrics and SkyWalking metrics.
+
+![PromQL Service architecture](img_1.png#pic_left)
+
+As SkyWalking is fundamentally different from Prometheus in terms of metric classification, format, storage, etc.
+The PromQL Service supported will be a subset of the full PromQL.
+Refer to the documentation for specific [compatibility](https://skywalking.apache.org/docs/main/next/en/api/promql-service/#promql-service).
+
+![PromQL Service compatibility](img.png#pic_left)
+
+## SkyWalking Basic Concepts
+Here are some basic concepts and differences from Prometheus that users need to understand in order to use the PromQL service:
+Prometheus metrics specify the naming format and structure, the actual metric names and labels are determined by the client provider, and the details are stored.
+The user aggregates and calculates the metrics using the expression in PromQL.
+Unlike Prometheus, SkyWalking's metric mechanism is built around the following core concepts with a hierarchical structure:
+- Layer: represents an abstract framework in computer science, such as Operating System(OS_LINUX layer), Kubernetes(k8s
+  layer). This layer would be the owner of different services detected from different technologies. All Layers definitions can be found [here](https://github.com/apache/skywalking/blob/master/oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/Layer.java).
+- Service: Represents a set/group of workloads which provides the same behaviors for incoming requests.
+- Service Instance: An individual workload in the Service group.
+- Endpoint: A path in a service for incoming requests.
+- Process: An operating system process. In some scenarios, a Service Instance is not a process, such as a pod Kubernetes could contain multiple processes.
```

Comment:
```suggestion
- Process: An operating system process. In some scenarios, a `service instance` is not a process, such as a pod Kubernetes could contain multiple processes.
```

I think you didn't capitalize the first letter in other parts of the doc. You should consider keeping them consistent.